strategies/failed/src/main/java/org/arquillian/smart/testing/strategies/failed/FailedTestsDetectorFactory.java

```diff
@@ -23,5 +24,9 @@ public TestExecutionPlanner create(File projectDir, TestVerifier verifier, Confi
         return new FailedTestsDetector();
     } // <2>
 
+    @Override
+    public StrategyConfiguration strategyConfiguration() {
+        return null;
```

Comment:
I'm not sure if this is the best thing for including into the documentation

Reply:
In general returning `null` is a bit odd, isn't it?

Reply:
then maybe I will create Empty Configuration class

Reply:
What would that solve? Why are we having this interface with only `name` method in it?

Reply:
> What would that solve?

Currently we don't have any configuration for `new`, `failed`, `affected` strategy for which we are returning null. May be instead of that Either create empty configuration so we can filter it out in `loadStrategyConfiguration()`

> Why are we having this interface with only name method in it?

If any strategy has configuration like affected (transitivity, inclusions, exclusions) then it can be defined by implementing `StrategyConfiguration`. In case of `affected` strategy we are defining [AffectedConfiguration](https://github.com/arquillian/smart-testing/pull/243/files#diff-94774a660c8fa42f85c66691e6db1fb8) which we are [loading](https://github.com/arquillian/smart-testing/pull/243/files#diff-348861c21efd28f5cc99117b2c67302aR161) after runtime dependency of strategy is added to classpath using `name()` method of `StrategyConfiguration`.

Reply:
Or maybe create configuration for each strategy with it's name having empty list of `ConfigurationItem` something like

```java
return new StrategyConfiguration() {
            @Override
            public String name() {
                return "failed";
            }

            @Override
            public List<ConfigurationItem> registerConfigurationItems() {
                return new ArrayList<>();
            }
        };
```